serow_ros/src/serow_driver.cpp

```diff
@@ -117,6 +117,7 @@ class SerowDriver {
                         static_cast<double>(joint_state_data.header.stamp.sec) +
                         static_cast<double>(joint_state_data.header.stamp.nsec) * 1e-9;
                     joint.position = joint_state_data.position[i];
+                    joint.velocity = joint_state_data.velocity[i];
```

Comment:
is this always available even when the `joint_state_data.velocity` field is empty?